persistent-template/ChangeLog.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased changes
 
+* Small optimization/code cleanup to generated Template Haskell code size, by slimming the implementation of to/fromPersistValue for Entities. [#1014](https://github.com/yesodweb/persistent/pull/1014)
```

Comment:
I'd be happy to release this as 2.8.0.1.

```suggestion

## 2.8.0.1

* Small optimization/code cleanup to generated Template Haskell code size, by slimming the implementation of to/fromPersistValue for Entities. [#1014](https://github.com/yesodweb/persistent/pull/1014)
```

Reply:
Ok, I think that's a good idea too. My other optimization effort failed, and I don't currently have more ideas